components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -159,7 +154,7 @@ const BillingSubscriptions = () => {
                                 onUpgradeMattermostCloud={onUpgradeMattermostCloud}
                             />
                         </div>
-                        {hasSomeLimits(cloudLimits) && !isFreeTrial ? (
+                        {product?.sku === CloudProducts.STARTER ? (
```

Comment:
We are calling a free trial if a user is using enterprise features, the starter plan is default and free, and that limit component is only valid for the starter pack now
 @neallred Please correct me; I have a wrong understanding

Reply:
@vish9812 good catch here! I think we want to leave the existing logic of `hasSomeLimits(cloudLimits) && !isFreeTrial`. We want this to be data driven rather than a hard-coded sku so that if in the future, other limits are added to professional plan, it still works without needing more changes.

I had to check on the `isFreeTrial` condition. Looks like we had an edge case where if someone was already on this page when they upgraded, sometimes it would show a blank card, and keeping the `!isFreeTrial` check was the fix for that:
* PR: https://github.com/mattermost/mattermost-webapp/pull/10595
* Ticket: https://mattermost.atlassian.net/browse/MM-45113